File: C/Exercicio2.c
//Algoritmo que soma dois números e multiplica o resultado pelo primeiro número.
#include <stdio.h> //std = standart io = input/output

int main() {
    //Variáveis
    int primeiroNum, segundoNum, soma, multiplicacao;

    //Entrada
    printf("Insira o primeiro número: \n");
    scanf("%d", &primeiroNum);
    
    printf("Insira o segundo número: \n");
    scanf("%d", &segundoNum);

    //Processamento
    soma = primeiroNum + segundoNum;

    multiplicacao = soma * primeiroNum;

    //Saída
    printf("O resultado da multiplicação dos dois números é: %d", multiplicacao);
    printf("\n");
    return 0;
}

File: Java/demo/src/main/java/com/exercicios/Exercicio2.java
//Algoritmo que soma dois números e multiplica o resultado pelo primeiro número.
package com.exercicios;

import java.util.Scanner;

public class Exercicio2 {
    public static void main(String[] args) {
        //Variáveis
        int num1, num2, soma, multiplicacao;
        Scanner entrada = new Scanner(System.in);

        //Entradas
        System.out.println("Informe o primeiro número: ");
        num1 = entrada.nextInt();
        System.out.println("Digite o segundo número: ");
        num2 = entrada.nextInt();

        //Processamento
        soma = num1 + num2;
        multiplicacao = soma * num1;

        //Saída
        System.out.println("O resultado da multiplicação é: " + multiplicacao);
        
        entrada.close();
    }
}
